### ISSUE TITLE
Reintroduce SCSS and modular structure

### ISSUE THREAD

**Issue by riede** [tobiasahlin/SpinKit#150]
With v2 you removed SCSS support. Now, only one big file with all spinner exists. But I only want to import the Pulse Spinner and the Three Bounce Spinner. This is currently not possible.

Please reintroduce SCSS and the modular structure (with SCSS variables), so I can only import the necessary spinner and setting.

**Comment by gigadie** [comment]
I totally agree on this one here. Used to compile locally only what I needed, in my own scss files, but now I am not able cause the scss support is removed.
It would be great to have it back in order to have the same flexibility we used to.
Thank you
